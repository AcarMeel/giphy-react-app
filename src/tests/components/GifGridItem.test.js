import React from 'react';
import { shallow } from "enzyme"
import GifGridItem from '../../components/GifGridItem';

describe('Tests in GifGridItem', () => {
    const title = 'Gif Title';
    const id = 1;
    const url = 'https://localhost:3000';
    test('should display component correctly', () => {
        const wrapper = shallow(<GifGridItem title={title} url={url} />);
        expect(wrapper).toMatchSnapshot();
    });
})
