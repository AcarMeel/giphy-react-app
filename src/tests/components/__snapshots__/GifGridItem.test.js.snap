// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in GifGridItem should display component correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <p>
    Gif Title
  </p>
  <img
    alt="Gif Title"
    src="https://localhost:3000"
  />
</div>
`;
